### PR TITLE
Feat/accept reject

### PR DESCRIPTION
**Pull request by zdtango**
I won't have time to complete this over the summer. I can return to it when things are less crazy, but here are my items left:
1. [x] styling
2. [x] adding a toggle for registrations
3. [x] implement a registration closed page given registrations are closed
4. [x] don't allow registrations at the registrations endpoint if registrations are closed
5. [x] geeth's ui magic

### CODE REVIEW

File: pages/admin/users.tsx [hunk 1]
```diff
@@ -38,23 +46,63 @@ export default function UserPage() {
...
+    }
+
+    for (const userData of usersData) {
+      userData.status = accepted.includes(userData.id)
+        ? 'Accepted'
+        : rejected.includes(userData.id)
+        ? 'Rejected'
+        : 'Waiting';
+      userData.selected = false;
+    }
```
Comment (DedsecKnight) [inline pages/admin/users.tsx]: Will need to re-implement this if we have like >= 1000 hackers because this is $O(n^2)$.

**Review by DedsecKnight — DISMISSED** [review]
Should be all good now, unless I'm missing something!

**Comment by abdullah50053** [comment]
👑  🚀

**Comment by abdullah50053** [comment]
A couple of comments:

Resumes are not visible in the application:
image

There's a bug where an unselected name can still be accepted:
https://github.com/acmutd/hackportal/assets/71054105/8d112885-e8b4-4eea-8f2b-5dc476c8c986

**Comment by DedsecKnight** [comment]
Should be all good now, although it doesn't hurt to have someone go over the whole registration process along with uploading their resume just in case the resume url is not logged into user's profile properly.

**Review by MichaelZhao21 — APPROVED** [review]

**Review by geeth24 — APPROVED** [review]

**Review by NickBurnett — APPROVED** [review]

**Pull request merged**